ops/pebble.py

```diff
@@ -460,7 +465,18 @@ def __init__(self, name: str, raw: Dict = None):
         self.after = list(raw.get('after', []))
         self.before = list(raw.get('before', []))
         self.requires = list(raw.get('requires', []))
-        self.environment = dict(raw.get('environment') or {})
+        self.environment = self._dicts_to_tuples(raw.get('environment') or [])
```

Comment:
Isn't that the same as: `raw.get('environment', [])` as we do with the other attributes?

Reply:
Not quite -- if the attribute is missing it'll use the `get` default, but if the attribute is present but explicitly `null`, `.get('environment', [])` will give `None`, where `.get('environment') or []` will give `[]`. Still, this should be consistent with the surrounding code, so I changed it. (In practice it should be a moot point, because Pebble won't give the fields explicitly set to `null`.